apps/api/Transcript Summarization.postman_collection.json

```diff
@@ -69,6 +69,53 @@
 				}
 			},
 			"response": []
+		},
```

Comment:
Add example with in operator using id.
Add it as GraphQL Query in description with response too.

Reply:
Added in both